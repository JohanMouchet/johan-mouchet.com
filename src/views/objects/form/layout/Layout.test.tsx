import React from "react";
import { shallow } from "enzyme";
import { Fieldset, FormGroup, Label } from ".";

it("Renders <Fieldset /> without crashing", () => {
  expect(shallow(<Fieldset />)).toMatchSnapshot();
});

it("Renders <FormGroup /> without crashing", () => {
  expect(shallow(<FormGroup />)).toMatchSnapshot();
});

it("Renders <Label /> without crashing", () => {
  expect(shallow(<Label />)).toMatchSnapshot();
});
