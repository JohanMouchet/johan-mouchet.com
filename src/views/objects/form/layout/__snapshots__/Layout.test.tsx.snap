// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Fieldset /> without crashing 1`] = `
<fieldset
  className="o-form__fieldset"
>
  <legend
    className="o-form__legend"
  >
    legend
  </legend>
  children
</fieldset>
`;

exports[`Renders <FormGroup /> without crashing 1`] = `
<div
  className="o-form__group o-form__group--success"
>
  children
  <span
    className="o-form__group-detail"
  >
    Detail
  </span>
</div>
`;

exports[`Renders <Label /> without crashing 1`] = `
<label
  className="o-form__label o-form__label--block"
>
  Label
</label>
`;
